src/oc/web/components/dashboard_layout.cljs

```diff
@@ -150,44 +119,89 @@
         topics (:topics board-data)]
       ;; Topic list
       [:div.dashboard-layout.group
-        (when (and @(::show-boards-tooltip s)
-                   (not show-onboard-overlay))
-          (when-let* [nav-boards (js/$ "h3#navigation-sidebar-boards")
-                      offset (.offset nav-boards)
-                      boards-left (aget offset "left")]
+        (when (= nux :3)
```

Comment:
I'd prefer the nux bits in its own component just to reduce the amount of code to read in `dashboard-layout`.